aes-kw/src/lib.rs

```diff
@@ -8,8 +9,41 @@
 #![forbid(unsafe_code)]
 #![warn(missing_docs, rust_2018_idioms)]
 
+//! # Usage
```

Comment:
Had to move the usage out of README.md in order to be able to feature-gate the doctests, unfortunately.

Otherwise they fail unless the correct features are available.